docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,128 @@
+With this approach, Tauri will follow the update server's instructions. To disable the internal version check you can [overwrite Tauri's version comparison] to always install the version sent by the server. This could be useful if you need to roll back your app version quickly.
 
-If no update is required your server must respond with a status code of [204 No Content].
+Your server can use variables defined in the `endpoint` url above to determine if an update is required. If you need more data, you can include additional [request headers in Rust] to your liking.
 
-### Update Server JSON Format
+Your server should respond with a status code of [`204 No Content`] if there is no update available.
 
-When an update is available, Tauri expects the following schema in response to the update request provided:
+If an update is required, your server should respond with a status code of [`200 OK`] and a JSON response in this format:
 
 ```json
 {
-  "url": "https://mycompany.example.com/myapp/releases/myrelease.tar.gz",
-  "version": "0.0.1",
-  "notes": "Theses are some release notes",
+  "version": "0.2.0",
   "pub_date": "2020-09-18T12:29:53+01:00",
-  "signature": ""
+  "url": "https://mycompany.example.com/myapp/releases/myrelease.tar.gz",
+  "signature": "Content of the relevant .sig file",
+  "notes": "These are some release notes"
 }
 ```
 
 The required keys are "url", "version" and "signature"; the others are optional.
 
-"pub_date" if present must be formatted according to [RFC 3339][date and time on the internet: timestamps].
-
-"signature" is the content of the `.sig` file which was generated by Tauri's CLI. See [Signing Updates](#signing-updates) for instructions on how to set up the required keys.
+- `"version"` must be a valid semver, with or without a leading `v`, meaning that both `1.0.0` and `v1.0.0` are valid.
+- `"url"` must be a valid url to the update bundle.
+- `"signature"` must be the **content** of the generated `.sig` file. The signature may change each time you run `tauri build` so make sure to always update it.
+- `"notes"`: Here you can add notes about the update, like release notes. Tauri's default dialog will present this to the user when it asks if it's allowed to update.
+- `"pub_date"` must be formatted according to [RFC 3339] if present.
 
-### Update File JSON Format
+### Static JSON File
 
-The alternate update technique uses a plain JSON file, storing your update metadata on S3, gist, or another static file store. Tauri checks against the version field, and if the version of the running process is smaller than the reported one of the JSON and the platform is available, it triggers an update. The format of this file is detailed below:
+With this approach, Tauri will always request the same JSON file and determine if the app needs to be updated by comparing the version field of the response with the current version. Tauri will expect a response in this format:
 
 ```json
```

Comment:
Again, is this format anywhere else or only here?

Reply:
Same as above.